sdk/src/main/java/com/ingresse/sdk/builders/URLBuilder.kt

```diff
@@ -15,15 +15,23 @@ enum class Environment(val prefix: String) {
     HML_A("hmla-"),
     HML_B("hmlb-"),
     TEST("test-"),
-    STG("stg-")
+    STG("stg-"),
+    INTEGRATION("integration2-")
+}
+
+enum class HostPrefix(val prefix: String) {
+    HTTP("http://"),
+    HTTPS("https://")
 }
 
 class URLBuilder(host: Host, environment: Environment = Environment.PROD) {
-    private val hostPrefix = "https://"
     private var parameters: MutableMap<String, String> = mutableMapOf()
     private var path = ""
     private var url: String
 
+    private val hostPrefix = (if (environment == Environment.INTEGRATION) HostPrefix.HTTP
```

Comment:
Aqui a gente não pode criar uma propriedade `HostPrefix` dentro do environment?

Assim a gente teria uma linha tipo: 
```suggestion
    private val hostPrefix = environment.hostPrefix
```

Reply:
Primeiro pensei em colocar isso num _When (Switch)_, porém temos somente dois tipos de _host prefix_, então pensei ao invés de duplicar cada um pra _enviroment_, apenas numa condição checar o tipo. O mesmo acontece num caso específico do enviroment mudar conforme host.

Acho que podemos conversar melhor sobre essa parte do SDK tanto Android quanto iOS e assim definirmos uma utilização bacana.

Reply:
Primeiro pensei em colocar isso num _When (Switch)_, porém temos somente dois tipos de _host prefix_, então pensei ao invés de duplicar cada um pra _enviroment_, apenas numa condição checar o tipo. O mesmo acontece num caso específico do _enviroment_ mudar conforme _host_.

Acho que podemos conversar melhor sobre essa parte do SDK tanto Android quanto iOS e assim definirmos uma utilização bacana.